internal/cocoa/ios.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Ebiten Authors
+// Copyright 2022 The Ebitengine Authors
```

Comment:
Sorry but I realized that `ios_darwin.go` is better so that non-Darwn environment cannot build this package.